ekncontent/ekncontent/eknc-subtree-dispatcher.c

```diff
@@ -191,12 +202,20 @@ const GDBusSubtreeVTable subtree_vtable = {
   .dispatch   = subtree_dispatch,
 };
 
+/**
+ * eknc_subtree_dispatcher_register:
+ * @dispatcher: the subtree dispatcher
+ * @connection: the dbus connection
+ * @subtree_path: a subtree path to register with
+ *
+ * Register the dispatcher.
+ */
 void
-ekns_subtree_dispatcher_register (EknsSubtreeDispatcher *dispatcher,
+eknc_subtree_dispatcher_register (EkncSubtreeDispatcher *dispatcher,
```

Comment:
I don't think it's an absolute rule but much of the other code uses it, so I usually flag it in reviews if I notice it